packages/modular-scripts/src/utils/selectWorkspaces.ts

```diff
@@ -68,14 +81,46 @@ export async function selectWorkspaces({
     packageScope = packageScope.concat([...ancestorsSet]);
   }
 
-  packageScope = [...new Set(packageScope)];
+  // We want to remove all the non-buildable packages from the package scope. Create a filter predicate that looks up to the package map
+  const isWorkspaceBuildable = (name: string) => {
+    const type = allWorkspacePackages.get(name)?.modular?.type;
+    return type && isBuildableModularType(type as PackageType);
+  };
+
+  packageScope = [...new Set(packageScope)].filter(isWorkspaceBuildable);
```

Comment:
Move this inside the if, or split the function in two. We need to select non-buildable packages in case we use this to select testable packages.